### ISSUE TITLE
Installing 5.2.2 from source is failing on macOS High Sierra

### ISSUE THREAD

**Issue by remomueller** [phusion/passenger#2054]
**Question 1: What is the problem?**
- What is the expected behavior?
- What is the actual behavior?
- How can we reproduce it? Please try to provide a sample application (or Virtual Machine) demonstrating the issue. Otherwise, if we can't reproduce it, we might have to ask you a number of followup questions or run certain commands to try and figure out the problem.

Be as detailed as possible in your descriptions, include any logs and stack traces (don't just cut/paste the error, provide some logging before that too).

(if you are requesting a feature instead of reporting an issue, describe here what you have in mind and how it would help you)

Your answer:

I'm installing passenger (5.2.2) + nginx from source using RVM (1.29.3) and Ruby (2.5.0) on MacOS. The installation does not complete and generates "It looks like something went wrong". I can install both previous versions 5.1.12. and 5.2.1 without this issue and the same setup. I've also checked that XCode is up to date and that the command line tools are installed. I have been able to install 5.2.2 using same setup on CentOS 6, so it appears to be Mac specific.

My installation that causes the error is:

```
gem install passenger --no-document
rvmsudo passenger-install-nginx-module --auto --prefix=/usr/local/nginx --languages ruby
```

The output is attached [in a gist here](https://gist.github.com/remomueller/71eae2aa2b002efd7eb342cebe3122c9).

**Question 2: Passenger version and integration mode**:
- For example: open source 5.0.26 standalone; enterprise 5.0.21/nginx

Your answer:

open source passenger 5.2.2 + default nginx 1.12.2

**Question 3: OS or Linux distro, platform (including version)**:
- For example: Debian 8, x86_64 or OS X 10.10 Yosemite, x86_64

Your answer:

MacOS 10.13.3 High Sierra

**Question 4: Passenger installation method**:

Your answer:
[ ] RubyGems + Gemfile
[X] RubyGems, no Gemfile
[ ] Phusion APT repo
[ ] Phusion YUM repo
[ ] OS X Homebrew
[ ] source tarball
[ ] Other, please specify:

Passenger installed with RubyGems, no Gemfile: installation
 [step 1](https://github.com/remomueller/documentation/blob/master/macos/160-passenger.md)
 [step 2](https://github.com/remomueller/documentation/blob/master/macos/170-install-nginx.md)

**Question 5: Your app's programming language (including any version managers) and framework (including versions)**:

Your answer: This is using Ruby 2.5.0 (RVM), Rails 5.2.0.rc2, Node.js 6.12.2

**Question 6: Are you using a PaaS and/or containerization? If so which one?**
- For example: Heroku, Amazon Container Services, Docker 1.9 with an image based on passenger-docker

Your answer: No, this is built on Mac OS (in development).

**Question 7: Anything else about your setup that we should know?**

Your answer: I tried various installing with extra modules first, then simplified it to using the simple approach above. I also tried uninstalling passenger completely and cleaning the gem files and downloaded source folders, and also rebooting to make sure it wasn't an error from something being cached.

**Comment by remomueller** [comment]
Looking into it a bit more, I'm wondering if the error is related to this commit: https://github.com/phusion/passenger/commit/26e23297883914e786f6f62a6fc7e604101f2f96

**Comment by remomueller** [comment]
Looks like this was fixed in 05cfddad83b58c0d56d86885dff4496fb90d8525 in 5.2.3 release, thanks!
